IFC4x3/PropertyConstants/p/PARALLEL_3lJJ0s6nr5eOwnkdiq2IaG.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="PARALLEL_3lJJ0s6nr5eOwnkdiq2IaG" Name="PARALLEL" UniqueId="ef4d3036-1b1d-45a1-8eb1-ba7b34092910" />
```

Comment:
Description.

Reply:
Fixed in [26b41fa](https://github.com/bSI-InfraRoom/IFC-Specification/pull/828/commits/26b41fa080bec029809d1e01e57a3b4e2c4d9122)